pkgs/development/tools/parsing/tree-sitter/default.nix

```diff
@@ -1,8 +1,10 @@
-{ lib, stdenv
+{ lib, stdenv, darwin
 , fetchgit, fetchFromGitHub, fetchurl
 , writeShellScript, runCommand, which
 , rustPlatform, jq, nix-prefetch-git, xe, curl, emscripten
 , callPackage
+, enableShared ? true
+, enableStatic ? false
```

Comment:
`enableShared` what? Not really clear what that means.

Do we need both options? Why is it configurable?

Reply:
If you're building static, maybe you want to not have the shared libraries - configurable by this flag

Reply:
I had forgotten to set `enableShared = false` in `static.nix`; fixed now